backend/src/openarchiefbeheer/api/urls.py

```diff
@@ -65,6 +66,7 @@
     basename="review-responses",
 )
 router.register(r"zaken", ZakenViewSet, basename="zaken")
+router.register(r"logs", LogsViewset, basename="logs")
```

Comment:
Based on what I see this the /destruction-lists/<uuid>/audit_log is replaced with a separate endpoint, this feels a bit less RESTful, is there a reason for this change?

Reply:
It's because the TimelineObjects are a different model. I feel that it makes more sense that it comes from its own endpoint. In this way we can also more easily use filtersets for filtering.